docs/stencil-docs/localization/localization-tutorial.md

```diff
@@ -0,0 +1,98 @@
+
+This article assumes you have familiarity with the following concepts:
+
+* [Installing Stencil CLI](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/installing-stencil)
+* [Creating a Stencil CLI Token](https://support.bigcommerce.com/s/article/Store-API-Accounts)
+* [Downloading and Uploading Custom Themes](https://support.bigcommerce.com/s/article/Stencil-Themes#download-upload)
+* [Serving a live preview](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/live-previewing-a-theme#serving-a-live-preview)
+
+### Prerequisites
+
+* For this tutorial, you will need to use Stencil CLI and use [Browsersyn](https://github.com/bigcommerce/browser-sync) to serve up a live preview of a theme in development.
+
+## Getting started
+1. Within the Stencil theme, go to the top-level `/lang` subdirectory. You will save your language files here. Each language requires its own JSON file. 
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+<!-- theme:  -->
+> **NOTE:** The `/lang` subdirectory includes the `/en.json` file. Both files must be present for a Stencil theme to work.
+
+</div>
+</div>
+</div>
+
+2. Use the `en.json` file to prepare your language file. Copy over the necessary key-value pairs and update the values.
+
+```json
+ ar.json                                           fr.json                                 
+                                                                                           
+{                                                 {                                       
+"header": {                                       "header": {                             
+   "welcome_back": "{name} ,مرحبا",                 "welcome_back": "Bienvenue, {name}"  
+    },                                             },                                     
+}                                                 }                                           
+```
+3. Name your translation file based on the [BCP 47 specification](https://tools.ietf.org/html/bcp47) of language and region codes.
+
+## Creating translation keys
+
+Perform the following steps to create new key-value pairs and invoke a defined translation key.
+
+1. Add a key-value pair to a language file.
+  
+  For example, in en.json, add `powered_by`.
+
+```html
+"footer": {
+        "title": "Footer Start",
+        "brands": "Popular Brands",
+        "navigate": "Site Navigate",
+        "info": "Info",
+        "categories": "Categories",
+        "call_us": "Call us at {phone_number}",
+        "powered_by": "Powered by"
+    },
+```
+2. Add the following syntax in the appropriate file.
+
+```html
+{{lang "translation.key" optionalVariable="someValue"}}
+```
+For this example, update footer.html as shown below.
+
+```html
+{{/if}} {{#if theme_settings.show_powered_by}}
```

Comment:
```
{{#if theme_settings.show_powered_by}}
  {{lang 'footer.powered_by'}} BigCommerce
{{/if}}
```
The example in the docs is invalid. You need to have closed if {{/if}}  after the open one {{#if}}